### DISCUSSION TITLE
How can I add nextMajorSchedule for specific branches only.

### DISCUSSION THREAD

**Discussion by narendra-evidi** [microsoft/AL-Go#995] [Q&A]
How can I add nextMajorSchedule for specific branches only. I tried with ConditionalSettings. But no luck. 

Can anyone give me an example to add nextMajorSchedule setting to  ConditionalSettings. Thanks.

**Comment by freddydk** [comment]
Currently, this scenario is not supported out of the box.

**Comment by freddydk** [comment]
GitHub actions only supports scheduled runs on the default branch.
We could create a small workflow, which triggers the other workflow on a specific branch - but it requires some coding.

**Reply by mazhelez** [reply]
Alternatively, for now, we can try change "Test Next Major" workflow to utilize the `workflowSchedule.includeBranches` setting (similar to "Update AL-Go System Files" workflow. Doc: https://github.com/microsoft/AL-Go/blob/main/RELEASENOTES.md#run-update-al-go-system-files-on-multiple-branches
